README.md

```diff
@@ -13,6 +13,18 @@ Running this command will do the following:
   3. Call `terraform import` for all the apps and update the `.tfstate`
   4. Using .tfstate, update main.tf to fill in the editable fields of the resource
 
+### Install - Requires Go
+Clone this repo then install using go.
+
+```sh
+go install github.com/onelogin/onelogin
+```
+
+### Install From Source - Requires Go
+clone this repository
```

Comment:
It actually says to clone this repo right here. 

Perhaps it could say to clone it in your gopath `~/go/src/github.com/onelogin`. I would expect a go user to know this though. 

Thoughts?

Reply:
Thats the install from source section. If you follow the existing standard install it doesnt work. So it looks like cloning the repo is required under both install options.

Reply:
These are effectively the same install paths. I think the first one was the leftover and should be removed.